### PR TITLE
CDAP-3611 Documenting Queue Debugger Tool

### PR DESCRIPTION
**Pull request by awholegunch**
Adds documentation of the Queue Debugger Tool to the Transaction Maintenance page.

Passes [Quick Build](http://builds.cask.co/browse/CDAP-DOB55-3. [Updated to Build 3]

Page Of Interest:

- [Operations: Transactions Maintenance](http://builds.cask.co/artifact/CDAP-DOB55/shared/build-3/Docs-HTML/3.3.0-SNAPSHOT/en/admin-manual/operations/tx-maintenance.html)

### CODE REVIEW

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst [hunk 1]
```diff
@@ -55,3 +58,42 @@ with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactio
 The current length of the invalid transaction list can be obtained using 
 :ref:`a call <http-restful-api-transactions-number>` 
 with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactions-number>`.
+
+
+Using the Queue Debugger Tool
+=============================
+The Queue Debugger Tool allows you to look at the entries in queues, and can be useful in
```
Comment (jwang47) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: `allows you to calculate queue statistics`

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst [hunk 2]
```diff
@@ -55,3 +58,42 @@ with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactio
...
+solving problems with transactions. This is a debug tool for a queue, returning
```
Comment (jwang47) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: `with queues`

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst [hunk 3]
```diff
@@ -55,3 +58,42 @@ with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactio
...
+information such as how many entries are in a queue, how many have been processed, and how
+many are in progress.
```
Comment (jwang47) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: `how many are not yet processed`

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst [hunk 4]
```diff
@@ -55,3 +58,42 @@ with the :ref:`Transaction Service HTTP RESTful API <http-restful-api-transactio
...
+
+  $ /etc/init.d/cdap-master run co.cask.cdap.data.tools.HBaseQueueDebugger
+  
+or::
+
+  $ /opt/cdap/master/bin/svc-master run co.cask.cdap.data.tools.HBaseQueueDebugger
+  
+Running the ``help`` option will give a summary of commands and required parameters.
+
+The tool ``SimpleHBaseQueueDebugger`` is a wrapper of the tool that uses common defaults.
```
Comment (jwang47) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: `that uses a set of defaults useful for displaying the minimum transaction time for all events in all queues.`

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: fixed.

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: fixed.

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst (hunk 3 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: fixed.

File: cdap-docs/admin-manual/source/operations/tx-maintenance.rst (hunk 4 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/operations/tx-maintenance.rst]: fixed.

**Comment by jwang47** [comment]
LGTM

**Pull request merged**